### ISSUE TITLE
Bug when updating club name

### ISSUE THREAD

**Issue by OysteinAmundsen** [OysteinAmundsen/gymsystems#113]
If a club has teams connected with it;

 ```
query failed: UPDATE `team` SET `club`=?  WHERE `id`=? -- PARAMETERS: [null,19]
error during executing query:Error: ER_BAD_NULL_ERROR: Column 'club' cannot be null
Error: ER_BAD_NULL_ERROR: Column 'club' cannot be null
     at Query.Sequence._packetToError (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Sequence.js:52:14)
     at Query.ErrorPacket (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
     at Protocol._parsePacket (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:279:23)
     at Parser.write (/usr/src/app/node_modules/mysql/lib/protocol/Parser.js:76:12)
     at Protocol.write (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:39:16)
     at Socket.<anonymous> (/usr/src/app/node_modules/mysql/lib/Connection.js:103:28)
     at emitOne (events.js:115:13)
     at Socket.emit (events.js:210:7)
     at addChunk (_stream_readable.js:266:12)
     at readableAddChunk (_stream_readable.js:253:11)
     --------------------
     at Protocol._enqueue (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)
     at PoolConnection.query (/usr/src/app/node_modules/mysql/lib/Connection.js:208:25)
     at /usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:214:49
     at Promise (<anonymous>)
     at MysqlQueryRunner.query (/usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:212:16)
     at MysqlQueryRunner.<anonymous> (/usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:266:51)
     at step (/usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:32:23)
     at Object.next (/usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:13:53)
     at /usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:7:71
     at Promise (<anonymous>)
"PUT /api/clubs/4 HTTP/1.0" 200 88 "https://www.gymsystems.org/configure/club/4/members" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36"
 
query failed: SELECT `contestant`.`id` AS `contestant_id`, `contestant`.`name` AS `contestant_name`, `contestant`.`birthYear` AS `contestant_birthYear`, `contestant`.`gender` AS `contestant_gender`, `contestant`.`club` AS `contestant_club`, `club`.`id` AS `club_id`, `club`.`name` AS `club_name` FROM `club_contestant` `contestant` INNER JOIN `club` `club` ON `club`.`id`=`contestant`.`club` WHERE `contestant`.`club` = ? -- PARAMETERS: [null]
 error during executing query:Error: ER_BAD_FIELD_ERROR: Unknown column 'NaN' in 'where clause'
Error: ER_BAD_FIELD_ERROR: Unknown column 'NaN' in 'where clause'
     at Query.Sequence._packetToError (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Sequence.js:52:14)
     at Query.ErrorPacket (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Query.js:77:18)
     at Protocol._parsePacket (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:279:23)
     at Parser.write (/usr/src/app/node_modules/mysql/lib/protocol/Parser.js:76:12)
     at Protocol.write (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:39:16)
     at Socket.<anonymous> (/usr/src/app/node_modules/mysql/lib/Connection.js:103:28)
     at emitOne (events.js:115:13)
     at Socket.emit (events.js:210:7)
     at addChunk (_stream_readable.js:266:12)
     at readableAddChunk (_stream_readable.js:253:11)
     --------------------
     at Protocol._enqueue (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:145:48)
     at PoolConnection.query (/usr/src/app/node_modules/mysql/lib/Connection.js:208:25)
     at /usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:214:49
     at Promise (<anonymous>)
     at MysqlQueryRunner.query (/usr/src/app/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:212:16)
     at QueryBuilder.<anonymous> (/usr/src/app/node_modules/typeorm/query-builder/QueryBuilder.js:728:58)
     at step (/usr/src/app/node_modules/typeorm/query-builder/QueryBuilder.js:32:23)
     at Object.next (/usr/src/app/node_modules/typeorm/query-builder/QueryBuilder.js:13:53)
     at fulfilled (/usr/src/app/node_modules/typeorm/query-builder/QueryBuilder.js:4:58)
     at <anonymous>
```

**Comment by OysteinAmundsen** [comment]
Seems this bug is only happening in production. Cannot seem to reproduce locally... :-(

**Comment by OysteinAmundsen** [comment]
Closed by #127
